lib/workers/repository/process/lookup/index.ts

```diff
@@ -384,6 +384,19 @@ export async function lookupUpdates(
           // TODO #7154
           update.newDigest =
             update.newDigest ?? (await getDigest(config, update.newValue))!;
+
+          // If the digest could not be determined, report this as otherwise the
+          // update will be omitted later on without notice.
+          if (update.newDigest === null) {
+            logger.debug(
+              { depName, ...update },
+              'Could not determine new digest for update.'
+            );
+            res.warnings.push({
+              message: `Could not determine new digest for update (${update.newValue}).`,
+              topic: depName,
+            });
```

Comment:
I'm not certain that this is always a "warnable" scenario. There might be cases where the digest lookup fails and that's expected/non-concerning, especially for the pinDigests scenario. Maybe only add this level of warning if `currentDigest` is set?

Reply:
Can you elaborate on scenarios where no digest computation is expected?

If we have a `currentDigest` or `pinDigests = true`, then in such cases, if the data source does not return a digest, Renovate silently discards updates. My understanding is that all of these cases would need a warning to avoid being stuck on older versions accidentally without noticing.

Reply:
but they would also not get any updates then, and the warning would help making that visible. To me, it sounds much better printing a warning, than doing nothing, skipping updates & assuming it might be okay. 

Happy to guard it with `currentDigest` though. You are definitely the expert here 😄

Reply:
Update as per your feedback.